### PR TITLE
LUN-103: Article choose exercise add plural to last option

### PR DESCRIPTION
**Pull request by ansamshtaiwi1**
This pull request belongs to an issue on our [bugtracker](https://issues.tuerantuer.org/secure/RapidBoard.jspa?rapidView=27&view=detail).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword _LUN_.

### CODE REVIEW

File: src/routes/choice-exercises/components/SingleChoiceListItem.tsx [hunk 1]
```diff
@@ -175,7 +175,7 @@ const SingleChoiceListItem = ({
         </StyledArticleText>
       </StyledArticleBox>
       <StyledWord selected={selected} pressed={pressed} correct={showCorrect} delayPassed={delayPassed}>
-        {word}
+        {word} {article.id === 4 ? "(plural)" : " "}
```
Comment (klinzo) [inline src/routes/choice-exercises/components/SingleChoiceListItem.tsx:178]: I think this needs some more work :)
The current way, there is always a space added and for articles different than `die (Plural)` even two spaces are added. There should only be a space if needed though, i.e. before the `(Plural)` text. Can you find a way to do so?

All string constants should be added to the `labels.json` in an appropriate place and used from there.

Also: I think your code style is a little bit off, we always use single quotes `'`. Are you using IntelliJ or something different like VSCode?

**Review by klinzo — CHANGES_REQUESTED** [review]

File: src/routes/choice-exercises/components/SingleChoiceListItem.tsx (hunk 1 above)
Comment (YasminNajar) [reply src/routes/choice-exercises/components/SingleChoiceListItem.tsx:178]: Hello, thank you for the feedback. For the first point, would "{word}{article.id === 4 && '(Plural)'}" solve the issue ?

Comment (klinzo) [reply src/routes/choice-exercises/components/SingleChoiceListItem.tsx:178]: Yes, I think so. However, I would not put the space into the labels.json. Not sure whether to only put `Plural` or `(Plural)`. Probably even the first one without brackets.

Comment (SteffiStuffel) [reply src/routes/choice-exercises/components/SingleChoiceListItem.tsx:178]: In the ticket it was explicitly defines as `(Plural)` with brackets.

Comment (klinzo) [reply src/routes/choice-exercises/components/SingleChoiceListItem.tsx:178]: Yes, but my point was whether to add it to labels.json with brackets or without. The user should always see brackets.

Comment (SteffiStuffel) [reply src/routes/choice-exercises/components/SingleChoiceListItem.tsx:178]: ahh, ok I get it. Would put it to labels.json without brackets, so we can reuse it if needed.

Comment (klinzo) [reply src/routes/choice-exercises/components/SingleChoiceListItem.tsx:178]: I agree :)

**Comment by YasminNajar** [comment]
Changes have been done, let me know if you want me to change anything else :)

File: src/constants/labels.json [hunk 2]
```diff
@@ -66,5 +66,8 @@
       "cancelExercise": "Übung beenden",
       "of": "von"
     }
+  },
+  "pluralArticle": {
+    "plurals": "Plural"
   }
```
Comment (SteffiStuffel) [inline src/constants/labels.json:72]: I think you can put this insede general and without nesting.

I suggest:

```
  "general": {
    "header": {
      "overview": "Übersicht",
      "overviewExercises": "Übersicht der Übungen",
      "cancelExercise": "Übung beenden",
      "of": "von"
    }
 "plural": "Plural"
  },
```

**Review by SteffiStuffel — APPROVED** [review]
Tested, looks good :tada:

**Pull request merged**